### ISSUE TITLE
Release of pre-trained model on C-10/100 and ImageNet

### ISSUE THREAD

**Issue by sunyiyou** [HobbitLong/SupContrast#94]
That would be best if you would like to release the pre-trained model, thanks!

**Comment by HobbitLong** [comment]
hi, the ImageNet pre-trained model I got is [here](https://www.dropbox.com/s/l4a69ececk4spdt/supcon.pth?dl=0). Be noted that this is **not** the original ImageNet model in the paper but it still gives you decent accuracy (e.g., 79.1 % top-1 accuracy with ResNet-50). The difference is that I train this model in PyTorch using the momentum encoder trick (a.k.a [MoCo](https://arxiv.org/abs/1911.05722)) with a smaller batch size of 1024.

For CIFAR models, I couldn't find them but they should be easily reproduced using the code in this repo.

**Comment by sunyiyou** [comment]
Thank you so much!
